app/jobs/check_fixity_recursive_job.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+class CheckFixityRecursiveJob < ApplicationJob
+  delegate :query_service, to: :metadata_adapter
+
+  def perform
+    file_set = query_service.custom_queries.least_recently_updated_file_set
```

Comment:
To reduce duplication, maybe we should just do `CheckFixityJob.perform_now(file_set.id)`?

Reply:
I think I was just avoiding loading the file_set object twice. but if that doesn't seem like an issue to you i'm glad to do it!

Reply:
Yeah, it only takes a couple dozen milliseconds to load - I don't mind doing it twice.